### DISCUSSION TITLE
Authenticator does not support the request.

### DISCUSSION THREAD

**Discussion by michaelt-94** [symfony/symfony#58167] [Q&A]
![Screenshot from 2024-09-03 16-38-20](https://github.com/user-attachments/assets/cf5d61de-a1de-43bb-9a1f-f3083086d96e)

I have created a simple site that has a home page and an admin page. I used the webapp make:user authentication as described in the documentation. Everything worked at that point. I had some non related OS issues so I decided to re-OS my machine. I pushed my working website to bitbucket. After re-OS my machine, I cloned my repo and performed composer install to add all the packages from composer.json.  Now, when I run symfony serve I get a blank white page at localhost:8000.
In the command terminal where server is running, I get this
[Application] Sep  3 19:41:38 |INFO   | REQUES Matched route "home_page". method="GET" request_uri="http://localhost:8000/" route="home_page" route_parameters={"_controller":"App\\Controller\\HomeController::index","_route":"home_page"}
[Application] Sep  3 19:41:38 |DEBUG  | SECURI Checking for authenticator support. authenticators=1 firewall_name="main"
[Application] Sep  3 19:41:38 |DEBUG  | SECURI Checking support on authenticator. authenticator="Symfony\\Component\\Security\\Http\\Authenticator\\FormLoginAuthenticator"
[Application] Sep  3 19:41:38 |DEBUG  | SECURI Authenticator does not support the request.
My home page does not have a FormLogin  login is located at /login.  I haven't made any changes. Can someone point me in the right direction to resolve this?
BTW, I can create a whole new symfony --webapp and it works just fine. So its something that happens when the repo is cloned.

I have done a search for similar issues, but I can find none that pertain to this happening on such a basic set up and after using git.

**Comment by MatTheCat** [comment]
> performed composer install to add all the packages from composer.json

Dependencies are supposed to be installed from composer.lock; see https://getcomposer.org/doc/01-basic-usage.md
If you really installed them from composer.json you may get dependencies incompatible with your project.

Regarding the `Authenticator does not support the request.` log this is normal: as you said your form login is one another page, so the `FormLoginAuthenticator` has nothing to do.

**Reply by michaelt-94** [reply]
OK, Thank you for your answer, HOWEVER, I am sorry I did not get out the Composer Bible or the Package Manager Encyclopedia in regard to my composer statement. So to be clear, I really don't care where Composer gets the data to load all the packages. All that matters is that the packages were loaded as expected when I cloned the repo so I could start my app again.

The point of explaining my composer steps and other steps that I took to clone my repo to get it started was USUALLY, someone will ask, "Did you do this? ...Did you do that?..." I was trying to avoid that.

So, to try and focus and to stay on point, regarding this statement, "Regarding the Authenticator does not support the request. log this is normal:"

No, this is not normal.

As I pointed out, I am trying to load "My Home Page" or "home_page" route.  There is absolutely NO custom code on this page in twig or the controller. This page has the base template from the bin/console setup. Why is it looking to the FormLoginAuthenticator?

IF you think this is normal for the FormLoginAuthneticator, then why is it normal for blank pages to be displayed?
